examples/gnrc_networking_rpl/Makefile

```diff
@@ -0,0 +1,93 @@
+# name of your application
+APPLICATION = gnrc_rpl_demo
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# we need to define the interface on which RPL will run
+ifeq (native,$(BOARD))
+  CFLAGS += -DCONFIG_GNRC_RPL_DEFAULT_NETIF=7
+else
+  CFLAGS += -DCONFIG_GNRC_RPL_DEFAULT_NETIF=6
+endif
```

Comment:
Apart from moving this to the original RPL example, what about other non-6LoWPAN or multi-interface boards? I think a smarter solution (until we have something better in general) would be to have this after

```Makefile
include $(RIOTBASE)/Makefile.include
```

and make it dependent on

1. if 6LoWPAN is included.
2. if more than one netdev driver is included (question is: which one to select then?).